openstack/barbican/templates/api-deployment.yaml

```diff
@@ -57,6 +57,15 @@ spec:
                   name: sentry
                   key: barbican.DSN.python
             {{- end }}
+            {{- if .Values.api.resources.enabled }}
```

Comment:
This will not work without a default value or some additional guards

Reply:
https://github.wdf.sap.corp/cc/secrets/blob/barbican_qa-de-2_values/staging/values/barbican.yaml#L35

https://github.wdf.sap.corp/cc/secrets/blob/barbican_qa-de-2_values/staging/values/barbican.yaml#L35

testing in qa-de-2 and staging at the moment